packages/extension-api/src/extension-api.d.ts

```diff
@@ -1385,4 +1385,51 @@ declare module '@podman-desktop/api' {
     export function saveImage(engineId: string, id: string, filename: string): Promise<void>;
     export const onEvent: Event<ContainerJSONEvent>;
   }
+
+  export interface AccountInfo {
+    id?: string;
+    label: string;
+  }
+
+  export interface AuthenticationSession {
+    idToken: string | undefined;
+    readonly id: string;
+    readonly accessToken: string;
+    readonly scopes: ReadonlyArray<string>;
+    readonly account: AccountInfo;
+  }
+
+  export interface AuthenicationSessionChangeEvent {
+    provider: AuthenticationProviderInfo;
+  }
+
+  export interface AuthenticationProviderSessionChangeEvent {
+    added?: AuthenticationSession[];
+    changed?: AuthenticationSession[];
+    removed?: AuthenticationSession[];
+  }
+
+  export interface AuthenticationProviderInfo {
+    readonly id: string;
+    readonly displayName: string;
+    readonly accounts: AccountInfo[];
+  }
```

Comment:
looks like it should not be used as `onDidChangeSessions` should return a `AuthenticationProviderAuthenticationSessionsChangeEvent` object and not a `AuthenticationProviderInfo`